client/app/components/parameters.html

```diff
@@ -6,6 +6,11 @@
       <input ng-switch-when="datetime-with-seconds" type="datetime-local" step="1" class="form-control" ng-model="param.ngModel">
       <input ng-switch-when="datetime-local" type="datetime-local" class="form-control" ng-model="param.ngModel">
       <input ng-switch-when="date" type="date" class="form-control" ng-model="param.ngModel">
+      <span ns-switch-when="enum">
+        <select ns-model="param.value" class="form-control">
+          <option ng-repeat="option in param.enumOptions.split('\n')" value="{{option}}">{{option}}</option>
```

Comment:
It's cleaner to do the split in the controller code (also gives more flexibility in the future).